views/participant.html.twig

```diff
@@ -96,7 +96,8 @@
                 {% endif %}
                 <br />
             </div>
-            <div class="status-block patient-status-block">
+            {% if app.isDVType is empty %}
```

Comment:
Since this method queries the sites table, put it in the controller and pass to the view.